src/component/Footer.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import React from 'react';
+import { FilterStatusType, Todo } from '../types/Todo';
+
+type Props = {
+  setFilterBy: (filter: FilterStatusType) => void;
+  todos: Todo[];
+  filterBy: FilterStatusType;
+};
+
+export const Footer: React.FC<Props> = ({ setFilterBy, todos, filterBy }) => {
+  const leftTodosCount = todos.filter(todo => !todo.completed).length;
```

Comment:
```suggestion
  const uncompletedTaskCount = todos.filter(todo => !todo.completed).length;
```
If you have left, probably there should be right as well 👀